src/Client/pages/WishList.fs

```diff
@@ -81,7 +82,7 @@ let update (msg:WishListMsg) model : Model*Cmd<WishListMsg> =
     | AddBook ->
         if Validation.verifyBook model.NewBook then
             let wishList = { model.WishList with Books = (model.NewBook :: model.WishList.Books) |> List.sortBy (fun b -> b.Title) }
-            { model with WishList = wishList; NewBook = Book.empty }, postWishListCmd(model.Token,wishList)
```

Comment:
mhm. Tbh I did not test many cases here. usually if some tester in my team is complaining, then I would now model this in a new message and make things explict